packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .AssetCard__asset {
-  height: calc(1rem * (253 / var(--rem-base)));
+  height: calc(1rem * (254 / var(--rem-base)));
```

Comment:
Where does this number come from? Adding a comment with an explanation would be good